backend/ttnn_visualizer/views.py

```diff
@@ -395,6 +395,34 @@ def get_profiler_performance_data(session: Instance):
         result = csv.get_all_entries(as_dict=True, limit=100)
         return jsonify(result)
 
+@api.route("/profiler/perf-results/list", methods=["POST"])
+def get_profiler_performance_data_list():
+    location = request.json.get("location")
+    if location == "remote":
```

Comment:
An alternative to the if/else block could be:

```suggestion
    config_key = "REMOTE_DATA_DIRECTORY" if location == "remote" else "LOCAL_DATA_DIRECTORY"
    report_directory = Path(current_app.config[config_key])
```